Sources/FluentUI_iOS/Components/Tab Bar/TabBarItem.swift

```diff
@@ -42,6 +42,9 @@ open class TabBarItem: NSObject {
         }
     }
 
+    /// Unique tabID for tab Bar item.
+    @objc public var tabId: String? = ""
```

Comment:
Seeing if I can make this edit in PR so we can complete...
```suggestion
    @objc public let tabId: String?
```